### PR TITLE
docs: update docs for less-loader@6

### PR DESCRIPTION
**Pull request by yoyo837**
### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / document update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link
#23596
close #23587

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |    Added supplementary code description for less-loader@6       |
| 🇨🇳 Chinese |    为less-loader@6  添加补充代码说明       |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️ 

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

-----
[View rendered docs/react/customize-theme.en-US.md](https://github.com/ant-design/ant-design/blob/less-loader@6-docs/docs/react/customize-theme.en-US.md)
[View rendered docs/react/customize-theme.zh-CN.md](https://github.com/ant-design/ant-design/blob/less-loader@6-docs/docs/react/customize-theme.zh-CN.md)
[View rendered docs/react/use-with-create-react-app.en-US.md](https://github.com/ant-design/ant-design/blob/less-loader@6-docs/docs/react/use-with-create-react-app.en-US.md)
[View rendered docs/react/use-with-create-react-app.zh-CN.md](https://github.com/ant-design/ant-design/blob/less-loader@6-docs/docs/react/use-with-create-react-app.zh-CN.md)

### CODE REVIEW

File: docs/react/customize-theme.en-US.md [hunk 1]
```diff
@@ -198,14 +207,22 @@ module.exports = {
...
++     // less-loader@6+ https://github.com/webpack-contrib/less-loader#lessoptions
++     // options: {
++     //  lessOptions: {
++     //    modifyVars: getThemeVariables({
++     //      dark: true, // 开启暗黑模式
++     //      compact: true, // 开启紧凑模式
++     //    }),
++     //    javascriptEnabled: true,
++     //  },
++     // },
```
Comment (afc163) [inline docs/react/customize-theme.en-US.md:225]: 这样太复杂了，保持 less-loader 6 的写法，在后面加个说明就行。

Comment (afc163) [reply docs/react/customize-theme.en-US.md:225]: 类似，可以再写详细一点。

image

Comment (yoyo837) [reply docs/react/customize-theme.en-US.md:225]: 那会不会又有小可爱不知道less-loader@5怎么写？对他们来说相当于不知道怎么定制主题。

Comment (afc163) [reply docs/react/customize-theme.en-US.md:225]: 给说明就行了，文档太复杂用户也会糊涂不知道怎么写。

Comment (yoyo837) [reply docs/react/customize-theme.en-US.md]: 改了下！

File: docs/react/customize-theme.en-US.md [hunk 2]
```diff
@@ -51,6 +51,7 @@ module.exports = {
     }, {
       loader: 'less-loader', // compiles Less to CSS
 +     options: {
++       // Should move less-specific options to the lessOptions after less-loader@6+
 +       lessOptions: {
 +         modifyVars: {
 +           'primary-color': '#1DA57A',
```
Comment (afc163) [inline docs/react/customize-theme.en-US.md:57]: 在代码后面加提示，`less-loader@5` 里把 `lessOptions` 这一级移除即可。

Comment (afc163) [reply docs/react/customize-theme.en-US.md:57]: 统一这样写好了：

```
`lessOptions` 的配置写法在 [less-loader@6.0.0](https://github.com/webpack-contrib/less-loader/releases/tag/v6.0.0) 里支持，如果你的 `less-loader` 版本还是 5，请升级或者去掉 `lessOptions` 这一级。
```

**Pull request merged**